### ISSUE TITLE
No way to set a Keyboard shortcut to focus on CodeGPT Chat's input bar?

### ISSUE THREAD

**Issue by utmishra** [JudiniLabs/code-gpt-docs#131]
Hi,

Great extension first of all 👏 

I am aware that you can set Keyboard shortcuts for certain actions in CodeGPT and I tried them as well:
<img width="624" alt="Screenshot 2023-04-23 at 14 09 41" src="https://user-images.githubusercontent.com/1212811/233838992-2a919f19-0693-45df-b347-e848e09c5af0.png">

As per the screenshots above, you can focus on CodeGPT Chat's sidebar view using a keyboard shortcut (set manually), however, the focus is not on the text input field, which should be the expected behaviour, as I would like to immediately jump to typing my request.

Currently, once the view is focused, I try to Tab my way into the input bar, but the tab-index seems to be inconsistent and more dependant on external elements as well, for some reason.

**Comment by d-mahard** [comment]
Yes please implement this one. This will greatly improve the experience, esp for those who wants to minimize the usage of mouse.

(btw, duplicate with #188 )
